.github/workflows/build.yml

```diff
@@ -24,7 +25,8 @@ jobs:
            cd build
            cmake .. \
             -DCMAKE_BUILD_TYPE=${{matrix.target}} \
-            -DCMAKE_CXX_COMPILER=${{matrix.compiler}}
+            -DCMAKE_CXX_COMPILER=${{matrix.compiler}} \
+            -DCMAKE_CXX_STANDARD=${{matrix.cxx_version}}
```

Comment:
I'm realizing I don't think this will actually propagate to the test binaries because they're using target_compile_features

Reply:
Yeah that was my concern I was having when I was asking about using `set(CMAKE_CXX_STANDARD 17)` instead. I'm not actually sure what is the "correct" manner to pass a feature flag to cmake through the CLI. Do you have any idea in this regard?

Reply:
I'm not actually sure. Probably it'll require adding an option and doing `target_compile_features(integration PUBLIC ${LIBASSERT_TESTING_STD})`